app.js

```diff
@@ -17,7 +17,11 @@ app.set('view engine', 'handlebars');
 app.use(express.urlencoded());
 app.use('/static', express.static('public'));
 
+// Routes.
+const discover = require('./routes/discover')
+
 app.get('/', (req, res) => res.render('index'));
+app.get('/discover', discover.view);
```

Comment:
Could you also add a unit test in https://github.com/googleinterns/step24-2020/blob/master/test/test-pages.js for verifing the routing and page content?